package.json

```
@@ -72,30 +72,35 @@
     "cross-env": "^7.0.0",
     "http-errors": "^1.7.3",
     "husky": "^4.2.1",
-    "jest": "^25.1.0",
+    "jest": "^26.1.0",
     "jest-junit": "^10.0.0",
     "lint-staged": "^10.0.2",
     "middy-middleware-json-error-handler": "^2.0.0",
-    "pkg-ok": "^2.3.1",
+    "pkg-ok": "^3.0.0",
     "prettier-standard": "^16.0.0",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.0",
-    "semantic-release": "^17.0.0",
+    "semantic-release": "^16.0.0",
     "serverless": "^1.37.1",
     "serverless-offline": "^6.1.1",
     "serverless-webpack": "^5.2.0",
     "source-map-support": "^0.5.10",
     "supertest": "^4.0.2",
-    "ts-jest": "^25.0.0",
-    "ts-loader": "^6.2.1",
-    "tslint": "^6.0.0",
+    "ts-jest": "^26.0.0",
+    "ts-loader": "^9.3.1",
+    "tslint": "^5.0.0",
     "tslint-config-prettier": "^1.18.0",
-    "tslint-consistent-codestyle": "^1.15.0",
+    "tslint-consistent-codestyle": "^1.16.0",
     "typedoc": "^0.17.0",
-    "typescript": "^3.1.2",
+    "typescript": "^4.6",
     "wait-on": "^4.0.0",
-    "webpack": "^4.37.0"
+    "webpack": "^5.75.0"
   },
   "repository": "git@github.com:dbartholomae/middy-middleware-class-validator.git",
-  "snyk": true
+  "snyk": true,
+  "lint-staged": {
+    "!src/*.spec.ts": [
+      "prettier-standard"
+    ]
+  }
```

Please do not change formatting rules.

```suggestion
  "snyk": true
  }
```

---

As I mentioned in the above conversation. This is also for solving the husky issue. I will commit with no verify and remove these changes.